doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,272 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing circuits
+==================
+
+PennyLane offers the :mod:`~.io` module to import quantum circuits and operations that were
+constructed outside of PennyLane. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
+`OpenQASM <https://docs.quantum.ibm.com/build/interoperate-qiskit-qasm2>`_, and `Quil
+<https://docs.rigetti.com/qcs/guides/quil>`_.
+
+.. note::
+
+    To import a quantum circuit defined using a particular framework, you will need to install the
+    corresponding PennyLane plugin for that framework. More information about PennyLane plugins is
+    available on the `plugins <https://pennylane.ai/plugins.html>`_ page.
+
+Importing quantum circuits
+--------------------------
+
+Importing quantum circuits can help you take advantage of PennyLane's various optimization,
+visualization, and interoperability features for existing circuits. For example, you can take a
+``QuantumCircuit`` from Qiskit, import it into PennyLane, and then apply a `circuit-cutting transform
+<https://pennylane.ai/qml/demos/tutorial_quantum_circuit_cutting/>`_ to reduce the number of qubits
+required to implement the circuit. You could also :ref:`compile <intro_ref_compile_circuits>` the
+circuit using `Catalyst <https://docs.pennylane.ai/projects/catalyst/en/stable/index.html>`__ or
+execute the circuit remotely using a supported hardware provider.
+
+Qiskit
+~~~~~~
+
+To import a quantum circuit from Qiskit, you must first install the `PennyLane-Qiskit
+<https://docs.pennylane.ai/projects/qiskit/en/stable/>`__ plugin. This can be done by running:
+
+.. code-block::
+
+    pip install pennylane-qiskit
+
+Now, suppose we define a Qiskit `QuantumCircuit
+<https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit>`__ as follows:
+
+.. code-block:: python
+
+    from qiskit import QuantumCircuit
+
+    qk_circuit = QuantumCircuit(2, 1)
+    qk_circuit.h(0)
+    qk_circuit.cx(0, 1)
+    qk_circuit.measure_all()
+
+We can convert the ``QuantumCircuit`` into a PennyLane quantum function using:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_qk = qml.from_qiskit(qk_circuit)
+
+Above, the :func:`~pennylane.from_qiskit` function converts a ``QuantumCircuit`` into a PennyLane
+template. This template can then be called from inside a QNode to generate a PennyLane circuit:
```

Comment:
Similar to the updated docs of `from_qiskit`, I'd rephrase to talk about this as a quantum function rather than a template.